Jellyfin.Server.Implementations/Migrations/20201004171403_AddMaxActiveSessions.cs

```diff
@@ -13,7 +13,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 name: "MaxActiveSessions",
                 schema: "jellyfin",
                 table: "Users",
-                nullable: true);
+                nullable: false);
```

Comment:
Set a default value?

Reply:
`default: 0`?

Reply:
I don't see an example in the existing migrations.

Reply:
Found it courtesy of StackOverflow, `defaultValue`.